filebrowser/run.sh

```diff
@@ -21,10 +21,21 @@ if [ "${DB_PATH}" != "filebrowser.db" ]; then
   DB_FLAG=" -d ${DB_PATH}"
 fi
 
+# Check if filebrowser db exists
+if [ ! -f ${DB_PATH} ]; then
```

Comment:
```suggestion
if [ -n "$DB_FLAG" ]; then
```
Ideally I'd want DB_FLAG to be an array, so it can handle spaces in DB_PATH, but for now let's at least improve this check to avoid syntax errors.

Reply:
This needs to be DB_PATH here. It's checking if the file exists so it can initialize it if it doesn't.